src/include/parser/query/graph_pattern/rel_pattern.h

```diff
@@ -28,7 +28,11 @@ class RelPattern : public NodePattern {
 
     inline std::string getUpperBound() const { return upperBound; }
 
-    inline ArrowDirection getDirection() const { return arrowDirection; }
+    inline ArrowDirection getDirection() const {
+        return arrowDirection == RIGHT ? ArrowDirection::RIGHT : ArrowDirection::LEFT;
+    }
+
+    inline bool isDirected() const { return arrowDirection != UNDIRECTED; }
```

Comment:
remove this function and just expose `getDirection() {return arrowDirection}`. Whoever calls this function can use arrowDirection == BOTH to check if it's undirected or not.